### PR TITLE
ticdc: update the docs about create changefeed

### PR DESCRIPTION
**Pull request by leoppro**
### What is changed, added or deleted? (Required)
update the docs about create changefeed

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -135,6 +135,9 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 
 - `--changefeed-id`: 同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
 - `--sink-uri`: 同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`/`pulsar`。
+- `--start-ts`: 指定 changefeed 的 start tso，TiCDC 集群将从这个 tso 开始拉取数据。
+- `--target-ts`: 指定 changefeed 的 target tso，TiCDC 集群将数据拉取到这个 tso 后自动停止。
```
Comment (amyangfei) [inline ticdc/manage-ticdc.md:139]: 可以说明一下不指定时候的默认值

Comment (leoppro) [reply ticdc/manage-ticdc.md:139]: ok

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -135,6 +135,9 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 
 - `--changefeed-id`: 同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
 - `--sink-uri`: 同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`/`pulsar`。
+- `--start-ts`: 指定 changefeed 的 start tso，TiCDC 集群将从这个 tso 开始拉取数据。
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:138]:
```suggestion
- `--start-ts`: 指定 changefeed 的 start tso，TiCDC 集群将从这个 tso 开始拉取数据，默认为当前时间。
```

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (leoppro) [inline ticdc/manage-ticdc.md:139]:
```suggestion
- `--target-ts`: 指定 changefeed 的 target tso，TiCDC 集群将数据拉取到这个 tso 后自动停止，默认为空，则 TiCDC 不会自动停止。
```

**Review by amyangfei — COMMENTED** [review]
LGTM

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -135,6 +135,9 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 
 - `--changefeed-id`: 同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
 - `--sink-uri`: 同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`/`pulsar`。
+- `--start-ts`: 指定 changefeed 的 start tso，TiCDC 集群将从这个 tso 开始拉取数据，默认为当前时间。
+- `--target-ts`: 指定 changefeed 的 target tso，TiCDC 集群将数据拉取到这个 tso 后自动停止，默认为空，则 TiCDC 不会自动停止。
+- `--config`: 指定 changefeed 配置文件。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:140]:
```suggestion
- `--start-ts`：指定 changefeed 的开始 TSO。TiCDC 集群将从这个 TSO 开始拉取数据。默认为当前时间。
- `--target-ts`：指定 changefeed 的目标 TSO。TiCDC 集群拉取数据直到这个 TSO 停止。默认为空，即 TiCDC 不会自动停止。
- `--config`：指定 changefeed 配置文件。
```

**Comment by TomShawn** [comment]
@amyangfei @leoppro  PTAL again, thanks~

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -135,6 +135,9 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 
 - `--changefeed-id`: 同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
 - `--sink-uri`: 同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`/`pulsar`。
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:137]:
```suggestion
- `--sink-uri`：同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`/`pulsar`。
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -135,6 +135,9 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 
 - `--changefeed-id`: 同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:136]:
```suggestion
- `--changefeed-id`：同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**